.coderrabbit.yml

```diff
@@ -0,0 +1,50 @@
+language: "en-US"
+reviews:
+  profile: "assertive"
+  request_changes_workflow: false
+  high_level_summary: true
+  poem: true
+  review_status: true
+  auto_review:
+    enabled: true
+tools:
+  spellcheck:
+    enabled: true
+  markdownlint:
+    enabled: true
+  biome:
+    enabled: true
+  ruff:
+    enabled: true
+  github-checks:
+    enabled: true
+    timeout_ms: 180000
+  languagetool:
+    enabled: true
+    enabled_only: false
+    level: default
+  hadolint:
+    enabled: true
+  yamllint:
+    enabled: true
+  gitleaks:
+    enabled: true
+  eslint:
+    enabled: true
+  actionlint:
+    enabled: true
+  semgrep:
+    enabled: true
+chat:
+  auto_reply: true
+  custom_messages:
+    greeting: "Thanks for the mention! I'll review your code shortly."
+    farewell: "I've completed the review. Let me know if you need any clarification!"
```

Comment:
@aakankshabhende I don't find these fields under coderabbit ai yaml configuration. Can you please confirm how it will help and work in the PR?

<img width="880" alt="image" src="https://github.com/user-attachments/assets/775893e9-de4c-4768-b853-d44a2f136e30">

Reply:
@akshatnema the coderrabbit bot requested these changes

Reply:
@aakankshabhende Can you please point that review message?

Reply:
https://github.com/asyncapi/website/pull/3298#discussion_r1812849045

Reply:
@akshatnema please check coderabbit bot's above comment